assets/js/blocks/checkout/inner-blocks/checkout-pickup-options-block/block.tsx

```diff
@@ -37,36 +43,44 @@ const getPickupAddress = (
 ): string => {
 	if ( option?.meta_data ) {
 		const match = option.meta_data.find(
-			( meta: MetaKeyValue ) => meta.key === 'pickup_address'
+			( meta ) => meta.key === 'pickup_address'
 		);
 		return match ? match.value : '';
 	}
 	return '';
 };
 
-/**
- * Renders a shipping rate control option.
- *
- * @param {Object} option Shipping Rate.
- */
-const renderShippingRatesControlOption = (
-	option: CartShippingPackageShippingRate
-): PackageRateOption => {
+const renderPickupLocation = (
+	option: CartShippingPackageShippingRate,
+	packageCount: number
+): RadioControlOption => {
 	const priceWithTaxes = getSetting( 'displayCartPricesIncludingTax', false )
-		? parseInt( option.price, 10 ) + parseInt( option.taxes, 10 )
-		: parseInt( option.price, 10 );
+		? option.price + option.taxes
+		: option.price;
 	const location = getPickupLocation( option );
 	const address = getPickupAddress( option );
 	return {
 		value: option.rate_id,
 		label: location
 			? decodeEntities( location )
 			: decodeEntities( option.name ),
-		secondaryLabel: (
-			<FormattedMonetaryAmount
-				currency={ getCurrencyFromPriceResponse( option ) }
-				value={ priceWithTaxes }
-			/>
+		secondaryLabel: createInterpolateElement(
+			/* translators: %1$s name of the product (ie: Sunglasses), %2$d number of units in the current cart package */
+			_n(
+				'<price/>',
+				'<price/> x <packageCount/> packages',
+				packageCount,
+				'woo-gutenberg-products-block'
+			),
```

Comment:
Could we please change this so that it doesn't use `_n()`? In non-English languages this behaves differently than a `if ( n == 1 ) ... else ...`
See also this being mentioned here: https://wordpress.org/support/topic/very-incorrect-use-of-_n-in-checkout-and-checkout-blocks/